plugins/package-managers/conan/src/main/kotlin/Conan.kt

```diff
@@ -227,13 +227,13 @@ class Conan(
     }
 
     private fun configureRemoteAuthentication(conanConfig: File?) {
-        // Install configuration from a local directory if available.
-        conanConfig?.let {
-            run("config", "install", it.absolutePath).requireSuccess()
-        }
-
-        // List configured remotes in "remotes.txt" format.
         val remoteList = runCatching {
+            // Install configuration from a local directory if available.
+            conanConfig?.let {
+                run("config", "install", it.absolutePath).requireSuccess()
+            }
+
+            // List configured remotes in "remotes.txt" format.
```

Comment:
commit-msg: Should it provide a rationale why failing to install the configuration can be tolerated?

Reply:
It's not really being tolerated (at least not more than a failure to list the remotes) as both cases will lead to a warning being logged in the `getOrElse` branch.